docs/vms/vms.md

```diff
@@ -122,6 +122,21 @@ Once done with Linux, shut down the VM and restore the parameter to its original
 $ xe vm-param-set uuid=VM-UUID platform:device_id=0002
 ```
 
+## 🐧Linux VMs
+### Old Linux kernels and XFS
+
+Performance issues have been observed after a live migration, under heavy disk activity, for VMs running kernels prior to v4.20 and using the XFS file system.
+
+XFS seems to have better performances with recent kernels but for older ones we recommend to use another journaled file system like EXT4.
+
```

Comment:
I think it would be appropriate to mention in which case we observed this (mainly RHEL 8-like with 4.18 kernel, since they default to XFS installation?)